nims-notes-things/src/TextArea.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const TextArea = (props) =>{
+
```

Comment:
remove this whitespace and add a space between the destructuring and the return statement